src/vcpkg/dependencies.cpp

```diff
@@ -472,14 +472,14 @@ namespace vcpkg::Dependencies
     }
     bool InstallPlanAction::has_package_abi() const
     {
-        if (!abi_info) return false;
-        return !abi_info.get()->package_abi.empty();
+        const auto p = abi_info.get();
+        return p && !p->package_abi.empty();
     }
     Optional<const std::string&> InstallPlanAction::package_abi() const
     {
-        if (!abi_info) return nullopt;
-        if (abi_info.get()->package_abi.empty()) return nullopt;
-        return abi_info.get()->package_abi;
+        const auto p = abi_info.get();
+        if (!p || p->package_abi.empty()) return nullopt;
+        return p->package_abi;
```

Comment:
Is there a reason these changes are here? They're fine, but they feel unrelated.

Reply:
I was getting rid of all the repeated "engaged" checks on optionals in this area